isotip.js

```diff
@@ -160,28 +152,30 @@ module.exports = {
         }
 
         var moTrigger = evt.target || evt.srcElement
+        var moTarget = self.selfOrAncestorByClass(moTrigger, 'tooltip-hover')
+        var relatedTarget = (moEvt.relatedTarget || moEvt.toElement)
 
-        if (self.hasClass(moTrigger)) {
+        // If the tooltip shouldn't autoclose, bail
+        if (self.currentTooltip && self.currentTooltip.getAttribute('data-autoclose') === 'false') {
           return
         }
 
-        // If the tooltip shouldn't autoclose, bail
-        if (self.currentTooltip && self.currentTooltip.getAttribute('data-autoclose') === 'false') {
+        if (relatedTarget && (moTarget === relatedTarget || self.hasParent(relatedTarget, moTarget))) {
```

Comment:
This `self.hasParent` could probably become `self.selfOrAncestorByClass(moTarget, relatedTarget)` if it was edited to take a node as well as a string